JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1427,4 +1447,68 @@ Descriptives <- function(jaspResults, dataset, options) {
     return(TRUE)
   else
     return(FALSE)
-}
\ No newline at end of file
+}
+
+.descriptivesStemAndLeaf <- function(container, dataset, variables, options) {
+
+  # parameters for graphics::stem
+  scale <- options[["stemAndLeafScale"]]
+  width <- 120   # we could add this, but the R documentation is unclear as to what it does.
+  atom  <- 1e-08 # we could add this, but the R documentation is unclear as to what it does.
```

Comment:
Why not add this as a default argument? Even if the R documentation is unclear as to what it does. That's what you do in .descriptivesStemAndLeafToTable anyways.